nucypher/blockchain/eth/deployers.py

```diff
@@ -197,14 +209,21 @@ class MinerEscrowDeployer(ContractDeployer):
     __upgradeable = True
     __proxy_deployer = DispatcherDeployer
 
-    def __init__(self, secret_hash, *args, **kwargs):
+    def __init__(self,
+                 secret_hash,
+                 economics: TokenEconomics = None,
+                 *args, **kwargs):
+
         super().__init__(*args, **kwargs)
         self.token_agent = NucypherTokenAgent(blockchain=self.blockchain)
         self.secret_hash = secret_hash
+        if not economics:
+            economics = TokenEconomics()
+        self.__economics = economics
 
     def __check_policy_manager(self):
         result = self.contract.functions.policyManager().call()
-        if result is constants.NULL_ADDRESS:
+        if result is self.blockchain.NULL_ADDRESS:
             raise RuntimeError("PolicyManager contract is not initialized.")
 
     def deploy(self) -> dict:
```

Comment:
In the docstring two lines below, it says `Deploy and publish the NuCypher Token contract`. It should be `Deploy and publish the MinersEscrow contract`

Reply:
:heavy_check_mark: